src/test/java/com/savi/geo/timezones/TzWhereTest.java

```diff
@@ -0,0 +1,126 @@
+package com.savi.geo.timezones;
+
+import org.junit.*;
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+
+public class TzWhereTest {
+
+    @Test
+    public void testSomePointGetsItsTimeZoneId() {
+        final double latitude  = 38.8044;
+        final double longitude = -77.0920;
+        final String actual = TzWhere.getLatLongTimeZoneId(latitude, longitude);
+        final String expected = "America/New_York";
+
+        assertThat(actual, equalTo(expected));
+    }
+
+    @Test
+    public void testPointInCollidedTileGetsRightTimeZone() {
+        // A point in an index tile(?) that maps to two timezone portions and
+        // needs resolution via the actual geometry (as of 2016-05):
+        // (37.952126 N / -86.764727) is in America/Indiana/Tell_City but
+        // America/Chicago also gets indexed into same internal tile.
+        final double latitude  = 37.952126;
+        final double longitude = -86.764727;
+        final String actual = TzWhere.getLatLongTimeZoneId(latitude, longitude);
+        final String expected = "America/Indiana/Tell_City";
+    }
+
+    @Test
+    public void testPointAtBoundaryVertex() {
+        // A point that is vertex on boundary polygon of Europe/Vatican:
+        // (Attempted exact values--but unconfirmed.)
+        final double latitude  = 41.9063606262207;
+        final double longitude = 12.45091629028328;
```

Comment:
> Wow, that's super precise.
It's just the same digits that got printed out when dumping the numbers. I don't recall if Java guarantees enough digits on output to get the same bits upon input.  It's not confirmed that the bits actually were the same, because it's not confirmed that two zones matched, but that might have been because Rome's boundary with Vatican City is a negative polygon, not a regular polygon.

Changed assertion to allow Rome too.